test/degreed/content/courses_test.rb

```diff
@@ -13,12 +13,17 @@ def test_create_a_course
               "content-type" => "application/json"
             },
             body: {
-              title: "New Course",
-              summary: "A Summary",
-              "external-id": "arstaroisen",
-              url: "https://dev.lessonly.com",
-              duration: 200,
-              "duration-type": "Seconds"
+              data: {
```

Comment:
I really botched this 😆 the docs clearly have the attributes nested like this: https://api.degreed.com/docs/#create-a-new-course